# see available mappings on src/maqui_joy/xbox.py

# pause button
b_pause: 'START'

b_tablet_reset: 'X'

b_photo: 'Y'
b_life: 'A'
b_tablet_logo: 'B'

# increment buttons
b_increment: 'UP'
b_decrement: 'DOWN'

# say phrase button
b_phrase_1: 'LEFT'
b_phrase_2: 'RIGHT'


b_behavior: 'LB'
# text: array of strings
tts_phrases: [
    " Hola !",
    " Hola !",
    " Me llamo Maqui",
    " Me llamo Maqui y soy un robot Pepper",
    " Trabajo en el laboratorio de robotica, \\pau=500\\ con mi amigo Bender y mi equipo Uchile Jom Breiquers",
    " Soy un robot de servicio, disegnado para acompagnar y ayudar a los humanos",
    "^start(animations/Stand/Emotions/Positive/Laugh_3) \\pau=1000\\ "
    ]

b_animation: 'RB'

animations: [
    "Stand/Waiting/Robot_1",
    "Stand/Waiting/SpaceShuttle_1",
    "Stand/Waiting/Zombie_1",
    'dance_saxo/guitar'
    ]